src/PrestissimoFileFetcher.php

```diff
@@ -43,7 +37,16 @@ protected function fetchWithPrestissimo($version, $destination) {
     array_walk($this->filenames, function ($filename) use ($version, $destination, &$requests) {
       $url = $this->getUri($filename, $version);
       $this->fs->ensureDirectoryExists($destination . '/' . dirname($filename));
-      $requests[] = new CopyRequest($url, $destination . '/' . $filename, false, $this->io, $this->config);
+      if ($this->progress) {
```

Comment:
@webflo Thanks for your review.
At line 65, the $filename variable is not defined. So should I remove lines 40 to 49 and use the previous line 46? And leave line 65 as it is?